Makefile

```
@@ -85,15 +81,12 @@ install-asus-shutdown: target/$(TARGET)/$(BIN_S)
 
-install-asusd_user: target/$(TARGET)/$(BIN_U)
-	$(INSTALL_PROGRAM) "./target/$(TARGET)/$(BIN_U)" "$(DESTDIR)$(bindir)/$(BIN_U)"
-
 install-rog_gui: target/$(TARGET)/$(BIN_ROG)
 	$(INSTALL_PROGRAM) "./target/$(TARGET)/$(BIN_ROG)" "$(DESTDIR)$(bindir)/$(BIN_ROG)"
 
-.PHONY: install-asusd install-asus-shutdown install-asusctl install-asusd_user install-rog_gui
+.PHONY: install-asusd install-asus-shutdown install-asusctl install-rog_gui
 
-install-program: install-asusd install-asus-shutdown install-asusctl install-asusd_user install-rog_gui
+install-program: install-asusd install-asus-shutdown install-asusctl install-rog_gui
```

checkmake (0.3.2)

[warning] Target "install-program" should be declared PHONY.

(phonydeclared)